backend/functions/src/emailing/routes.ts

```diff
@@ -24,20 +31,45 @@ const router = express()
       FAIL -> res.data = 'Email send failure.' 
 
     */
-router.post('/send', (req, res) => {
+router.post('/send', async (req, res) => {
   const {
     emailAddress,
     authToken,
     emailBody,
     emailSubject,
-    emailRcpts,
     courseId,
     group,
     template,
+    indivEmail,
   } = req.body
+
+  let emailRcpts = ['lscstudypartners@cornell.edu']
+
+  if (parseInt(group) > 0) {
+    const groupData = await courseRef
+      .doc(courseId)
+      .collection('groups')
+      .doc(group)
+      .get()
+
+    emailRcpts = [...emailRcpts, ...(groupData.data() as any).members]
+  }
+
+  if (indivEmail !== undefined) {
+    emailRcpts = [...emailRcpts, indivEmail]
+  }
```

Comment:
It might be worth putting this and the guard code in its own function so it would be easier to unit test if we later add backend testing. In other parts of the backend the callback functions given to `router.post` and similar are limited to only extracting data from the request `req` and passing it off to the main function, which may throw errors if it has problems or the input is invalid. The `.then` and `.catch` functions attached to the function call in this area of code handle sending the frontend the proper response, which has a success or failure HTTP error code attached.